components/dialpad/tsconfig.json

```
@@ -0,0 +1,20 @@
+{
+    "compilerOptions": {
+      "lib": ["es2020"],
+      "module": "ES2020",
+      "target": "ES2020",
+      "moduleResolution": "node",
+      "listEmittedFiles": true, // Used as a part of the build task, since we need to pass emitted files to our post-build script
+      "composite": true,
+      "outDir": "dist",
+      "allowSyntheticDefaultImports": true,
+    },
+    "allowJs": true,
+    "include": [
+      "app",
+      "sources"
```

Since the `common` directory also contains `.ts` files that should be included in the TypeScript project and compiled, add `"common"` to `include`:
```suggestion
      "sources",
      "common"
```